README.md

```diff
@@ -66,7 +66,7 @@ $ info.keywords
 ```
 
 There is also in interactive repl mode: `history`, `!23` to repeat the 23rd statement for instance and `help`.
-And full edit and history navigation with arrow keys etc...
+And full edit and history navigation with arrow keys etc... and hit the `<tab>` key at anytime for completion.
```

Comment:
Two and in the same sentence.

It sounds strange, especially when previous one has an and

You could use a list like this

 There is also in interactive repl mode:
- `history`
- `!23` to repeat the 23rd statement for instance
- `help`. 
- full edit and history navigation with arrow keys
-  hit the `<tab>` key at anytime for completion.
- etc...

Reply:
thanks, good suggestion! done